src/core/style/_variables.scss

```diff
@@ -3,6 +3,10 @@
 
 // Typography
 $md-body-font-size-base: rem(1.400) !default;
+$font-family: Roboto, 'Helvetica Neue', sans-serif !default;
+
+// Media queries
+$md-mobile-screen: 600px;
```

Comment:
I wouldn't call this "mobile", instead using the breakpoint names from the spec (xs, small, etc.)